lib/mix/tasks/firmware.ex

```diff
@@ -296,4 +288,31 @@ defmodule Mix.Tasks.Firmware do
       """)
     end
   end
+
+  defp config_arg(:rootfs_overlay, firmware_config) do
```

Comment:
```suggestion
  defp rootfs_overlay_arg(firmware_config) do
```

LGTM! Though I would probably suggest descriptive purpose built functions for these args rather than a generic `config_arg` that has variable key arg

Reply:
I liked the single function where all the config keys were exposed to find. But since I had to reduce complexity for the cyclomat I wanted to do something that made it visually simpler but kept the keys visible. Okay to do the suggested change but it becomes a bit of a hunt to figure out what's needed or involved when diving in I think.